src/lib/core/data-table/data-table.ts

```diff
@@ -92,9 +98,13 @@ export class CdkTable implements CollectionViewer {
   /** Set of templates that used as the data row containers. */
   @ContentChildren(CdkRowDef) _rowDefinitions: QueryList<CdkRowDef>;
 
-  constructor(private _changeDetectorRef: ChangeDetectorRef) {
+  constructor(private _differs: IterableDiffers,
+              private _changeDetectorRef: ChangeDetectorRef) {
```

Comment:
I don't think we've set that precedent, but I can talk with Jeremy about adding it to here and other components as well